cupy/statistics/correlation.py

```diff
@@ -1,10 +1,109 @@
-# flake8: NOQA
-# "flake8: NOQA" to suppress warning "H104  File contains nothing but comments"
+import warnings
 
-# TODO(okuta): Implement corrcoef
+import numpy
+
+import cupy
+from cupy import core
+
+
+def corrcoef(a, y=None, rowvar=True, bias=numpy._NoValue, ddof=numpy._NoValue):
```

Comment:
`numpy._NoValue` is moved to `numpy._globals.NoValue` now.  (see the latest manual https://docs.scipy.org/doc/numpy/reference/generated/numpy.corrcoef.html)
I think it is OK to just use `None` because:
- A user usually does not use `_NoValue`. This value is given usually when it is omitted.
- These values are not used now.